src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -93,6 +94,8 @@ def get_validated_dmatrices(train_path, validate_path, content_type, csv_weights
         if train_path == validate_path or os.path.basename(train_path) == os.path.basename(validate_path):
             logger.warning('Found same path for training and validation. This is not recommended and results may not '
                            'be correct.')
+        else:
```

Comment:
for readability maybe put a comment here? that this just warns cx for now, no conditional logic happens here

Reply:
updated